base/base/wide_integer_impl.h

```diff
@@ -32,6 +34,59 @@ namespace CityHash_v1_0_2 { struct uint128; }
 namespace wide
 {
 
+constexpr bool supportsBitInt256()
+{
+#if defined(__x86_64__) && defined(__clang__) && (__clang_major__ > 14 || (__clang_major__ == 14 && __clang_minor__ >= 0))
```

Comment:
Which simply means "yes", because we only use clang and we use minimum clang-19 as of today.